journal/modernized/02-chapter-1.adoc

```diff
@@ -910,9 +910,9 @@ these Friends and William Edmundson met with much personal abuse.--See Rutty's _
 In the year 1676, John Tiffin,
 accompanied John Burnyeat through Westmoreland and Yorkshire, to London;
 and in 1684, into the north of England and Scotland.
-See Whiting''s Memoirs.
+See Whiting'`'`s Memoirs.
```

Comment:
This was an error in document, but thought you might want to see how it rendered it. Looks like it took the two apostrophes and turned the both the wrong way.